kuma/wiki/api/v1/views.py

```diff
@@ -5,23 +5,13 @@
 
 from kuma.wiki.models import Document
 
-
-@never_cache
-@require_GET
-def doc(request, locale, slug):
-    """
-    Return a JSON object that includes document content and metadata
-    for the document specified by the locale and path. Raises a 404
-    error if no such document exists. This is an API with URL
-    /api/v1/doc/<locale>/<path>
-    """
-    document = get_object_or_404(Document, locale=locale, slug=slug)
+def document_api_data(document):
```

Comment:
Most of what was previously the body of the doc() request handler below is now in this utility document_api_data() function. I made this change so that I can also get the json blob in kuma/wiki/views/document.py